library/core/tests/atomic.rs

```diff
@@ -127,6 +127,91 @@ fn int_max() {
     assert_eq!(x.load(SeqCst), 0xf731);
 }
 
+#[test]
+#[cfg(any(not(target_arch = "arm"), target_os = "linux"))] // Missing intrinsic in compiler-builtins
```

Comment:
Why is this specific to Linux? I would expect the same behaviour e.g. for FreeBSD.

Reply:
I don't know, I just copied this cfg from the other tests in the file.

That said, arm linux has special atomic support: https://github.com/rust-lang/compiler-builtins/blob/master/src/arm_linux.rs, so I'd assume it's related to that.

Reply:
Marking this as resolved because it doesn't seem actionable, and if we change something here it should be done for all the tests in this file (which should be done as a separate PR).